mos-platform/atari8-mega/link.ld

```diff
@@ -0,0 +1,209 @@
+/* Atari800 MegaCart and SIC! (bank-switched) cartridge linker script.
+
+  Support 16 KiB to 512 KiB MegaCart and SIC! cartridges.  These bank
+  the entire 16 KiB address space and also allow for disabling the
+  cartridge to access underlying RAM.
+
+  There are log2(__cart_rom_size) switchable banks of 16 KiB size
```

Comment:
Is this comment right? `__cart_rom_size` is in kilobytes... so I'd guess it's `__cart_rom_size / 16` banks, and `log2(__cart_rom_size / 16)` bits.

Reply:
Thanks, it's completely wrong, I'm not sure what I was thinking.